subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ShadowPluginSmokeTest.groovy

```diff
@@ -48,15 +51,19 @@ class ShadowPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
             """.stripIndent()
 
         when:
-        def result = runner('shadowJar').build()
+        def result = runnerWithExpectedDeprecationWarning('shadowJar').build()
```

Comment:
:question: Emitting a deprecation for the latest version of the Shadow plugin is more concerning, as usual.

* I opened https://github.com/johnrengelman/shadow/pull/821

I think if we hear back from John we can proceed.

Reply:
Cool, thanks for creating a PR!

Reply:
Well, that plugin is looking for a maintainer so :roll_eyes: